packages/gatsby/src/utils/get-ssl-cert.js

```diff
@@ -46,10 +71,19 @@ module.exports = async ({ name, certFile, keyFile, directory }) => {
       const mkdtemp = fs.mkdtempSync(path.join(os.tmpdir(), `home-`))
       process.env.HOME = mkdtemp
     }
-    const certificateFor = require(`devcert`).certificateFor
-    return await certificateFor(name, {
-      installCertutil: true,
+    const getDevCert = require(`devcert`).certificateFor
+    const { caPath, key, cert } = await getDevCert(name, {
+      getCaPath: true,
+      skipCertutilInstall: false,
+      ui: {
+        getWindowsEncryptionPassword,
+      },
     })
+    if (caPath) process.env.NODE_EXTRA_CA_CERTS = caPath
```

Comment:
Nitpick, no big deal but rather wrap it in a {}. We should add an eslint rule for it.
```suggestion
    if (caPath) {
      process.env.NODE_EXTRA_CA_CERTS = caPath
    }
```